src/campaigns/swapPromoCampaign.ts

```diff
@@ -23,16 +28,21 @@ export const swapsCampaignAction = async () => {
   mmkv.set(CampaignKey.swapsLaunch, true);
   setTimeout(() => {
     logger.log('triggering swap promo action');
+    analytics.track('Presented Feature Promo', { campaign: 'swaps_launch' });
```

Comment:
we should move this into the promo sheet so its a reusable event

Reply:
oops its already there, this is a leftover